File: URI/1 - INICIANTE/C/1101 - SequenciaDeNumerosESoma.c
#include <stdio.h>

int main()
{
	int x = 0, soma, m,n, max, min, y;
	while(x == 0)
	{
		scanf(" %d %d", &m, &n);
		if(m > n)
		{
			max = m;
			min = n;
		}else{
			max = n;
			min = m;
		}
		soma = 0;
		if (m > 0 && n > 0)
		{	
			for(y = min; y <= max; y++){
				printf("%d ", y);
				soma += y;
			}	
			printf("Sum=%d\n",soma);
		}else{
			x = 1;
			break;
		}	
	}
	
	
	return 0;
}

File: URI/1 - INICIANTE/Python/1101 - SequenciaDeNumerosESoma.py
l = 0
while l == 0:
    m, n = input().split('')
    m = int(m)
    n = int(n)
    soma = 0
    if m > 0 and n > 0:
        for y in range(min(m,n), max(m,n)+1):
            print(y, end=" ")
            soma += y
        print("Sum={}".format(soma))
    else:
        l = 1
        break